sda/cmd/download/swagger_v1.yml

```diff
@@ -0,0 +1,265 @@
+        "500":
+          description: Internal application error
+      security:
+        - bearerAuth: []
+  /info/datasets/{datasetId}:
+    get:
+      description: Returns an array with metadata about the dataset
+      parameters:
+        - in: header
+          description: "Public key used to re-encrypt the file header with"
+          name: public_key
+          required: true
+          schema:
+            type: string
+        - in: header
+          description: "The JWT used to authenticate the user"
+          name: token
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: datasetId
+          schema:
+            type: string
+          required: true
+          description: Stable ID of the dataset of interest
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DatasetInfo"
+          description: Successful operation
+        "401":
+          description: Authentication failure
+        "403":
+          description: The user does not have access to the dataset
+        "500":
+          description: Internal application error
+      security:
+        - bearerAuth: []
+  /info/datasets/{datasetId}/files:
+    get:
+      description: Returns an array with metadata about all files in a dataset
+      parameters:
+        - in: header
+          description: "Public key used to re-encrypt the file header with"
+          name: public_key
+          required: true
+          schema:
+            type: string
+        - in: header
+          description: "The JWT used to authenticate the user"
+          name: token
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: datasetId
+          schema:
+            type: string
+          required: true
+          description: Stable ID of the dataset of interest
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/FileInfo"
+          description: Successful operation
+        "401":
+          description: Authentication failure
+        "403":
+          description: The user does not have access to the dataset
+        "500":
+          description: Internal application error
+      security:
+        - bearerAuth: []
+  /ready:
+    get:
+      description: Returns the status of the application, used by the Kubernets API to decide if traffic should be routed to the app
+      responses:
+        "200":
+          description: The service is operational
+        "503":
+          description: Unhealthy service
+components:
+  schemas:
+    DatasetInfo:
+      type: object
+      properties:
+        date:
+          type: string
+          format: date-time
+          example: "2025-02-14T14:51:26.639Z"
+        files:
```

Comment:
`files` represents number of files, right? Then maybe to use a clearer name, such as `fileCount`.